From f7a0409730c486cc4151be60537ca71c25e2bf69 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 13 Nov 2020 02:09:36 +0000
Subject: [PATCH 1/5] LOCAL: configs: meson64: prevent stdout/stderr on
 videoconsole

Several devices have CONFIG_VIDEO enabled which causes stdout/stderr
to appear on videoconsole, so remove videoconsole from STDOUT so that
early u-boot boot remains silent unless using the uart/serial console.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index efab9a624d..0976774e5c 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -28,7 +28,7 @@
 
 /* For splashscreen */
 #ifdef CONFIG_VIDEO
-#define STDOUT_CFG "vidconsole,serial"
+#define STDOUT_CFG "serial"
 #else
 #define STDOUT_CFG "serial"
 #endif
-- 
2.34.1

